modules/virt-installing-virtio-drivers-installing-windows.adoc

```diff
@@ -23,7 +20,7 @@ documentation for the version of Windows that you are installing.
 
 .Procedure
 
-. In the Windows Guest OS, use the `File Explorer` to navigate to the `virtio-win` CD drive.
+. In the Windows operating system, use the `File Explorer` to navigate to the `virtio-win` CD drive.
 . Double-click the drive to run the appropriate installer for your VM:
```

Comment:
Does this need a `:` and `+` here? Seems like this and the line after the `+` could be together. Maybe there were previously more options here where the `:` made more sense.